Lib/slapdtest/_slapdtest.py

```diff
@@ -109,6 +110,35 @@ def requires_ldapi():
         return identity
 
 
+def _which(cmd):
+    """Specialized which command based on shutil.which() from Python 3.6.
+
+    * simplified
+    * always adds /sbin directories to path
+    """
+    path = os.environ.get("PATH", os.defpath).split(os.pathsep)
+
+    if sys.platform == 'win32':
+        if os.curdir not in path:
+            path.insert(0, os.curdir)
+        # include path extension (.exe)
+        pathext = os.environ.get("PATHEXT", "").split(os.pathsep)
+        files = [cmd + ext for ext in pathext]
+    else:
+        # always include sbin for slapd binary
+        for sbin in ['/sbin', '/usr/sbin', '/usr/local/sbin']:
+            if sbin not in path:
+                path.append(sbin)
+        files = [cmd]
+
+    for directory in path:
+        for name in files:
+            name = os.path.join(directory, name)
+            if os.path.exists(name) and os.access(name, os.F_OK | os.X_OK):
+                return name
+    return None
+
+
```

Comment:
Here too, I feel it would be nice to avoid auto-discovery if `CI` is set.

Reply:
What do you mean by auto-detection here? ``_which`` simply looks through PATH.